Objects/boolobject.c

```diff
@@ -110,9 +110,10 @@ bool_xor(PyObject *a, PyObject *b)
 /* Doc string */
 
 PyDoc_STRVAR(bool_doc,
-"bool(x) -> bool\n\
+"bool(object=False, /)\n\
```

Comment:
Sphinx docs shows this as ``bool(x=False)`` (give impression that this is a positional-or-keyword argument).  Probably, this should be adjusted too?

Reply:
Maybe in other issue. The documentation not always uses syntax for positional-only parameters. It is relatively new.